cms/templates/import.html

```diff
@@ -48,11 +72,23 @@ <h3 class="title">
 
           <input type="file" name="course-data" class="file-input" />
 
-          <input type="submit" value="${_('Replace my course with the one above')}" class="submit-button" />
+          <input type="submit"
+          %if library:
+              value="${_('Replace my library with the one above')}"
+          %else:
+              value="${_('Replace my course with the one above')}"
```

Comment:
Equivalent suggestion for importing courses: "Replace my course with the selected file"